### PR TITLE
Runnable as npm module.

### PR DESCRIPTION
**Pull request by JarnoLeConte**
I tried to make this code also work as node module instead of CLI only. I can't verify if all intended behaviour is preserved, because tests are missing. But it work well in my project at least :)
Maybe some improvements are necessary, but it looks promising.

I used it in the project `meteor-electron` to automatically package meteor desktop applications. See pull request https://github.com/electron-webapps/meteor-electron/pull/86

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
         {{> css}}
         {{> scripts}}
         {{> head}}
+        <meta charset="UTF-8">
```
Comment (JarnoLeConte) [inline index.html]: Without this, special characters are not preserved in my test case.

File: cli.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    // start building the meteor client
+    meteorBuildClient({
+        input: program.input,
+        output: argPath,
+        template: program.template,
+        settings: program.settings,
+        url: program.url,
+        path: program.path,
+        feedback: true,
```
Comment (JarnoLeConte) [inline cli.js]: feedback=true will output console messages.

File: meteor.js [hunk 3]
```diff
@@ -50,38 +48,39 @@ var deleteFolderRecursive = function(path) {
...
-
+    move: function(config, callback){
         try {
             _.each([
                 '/bundle/programs/web.browser',
                 '/bundle/programs/web.browser/app'
             ], function(givenPath){
-                var clientPath = path.join(buildPath, givenPath);
+                var clientPath = path.join(config.output, givenPath);
+                if (!fs.existsSync(clientPath)) return;
```
Comment (JarnoLeConte) [inline meteor.js]: Added this check, because the `app` folder isn't always present.

File: cli.js (hunk 2 above)
Comment (JarnoLeConte) [inline cli.js]: feedback=true will output console messages.

**Comment by frozeman** [comment]
Sorry for not responding earlier. I found this PR just now. Its hard for me to see whats the differences as you renamed files asfaik.
Could you rebase your changes on the latest develop. @hiddentao made some changes to the way meteor build client processes internally, with adding bluebird promises.

File: index.html (hunk 1 above)
Comment (adamgedney) [reply index.html]: I had the same issue. I saw it in FontAwesome rendering strange characters instead of Unicode.

**Comment by dr-dimitru** [comment]
Closing in favor of [`v1.0.0`](https://github.com/frozeman/meteor-build-client/releases/tag/1.0.0).
Feel free to reopen or send another PR if issue not solved in v1

**Pull request closed without merging**